lib/cn/cn_tree.c

```diff
@@ -1732,8 +1660,13 @@ cn_comp_commit(struct cn_compaction_work *w)
     }
     txn_nak = true;
 
-    if (split)
-        cn_split_nodeids_get(w, nodeidv);
+    if (split) {
+        w->cw_err = cn_split_nodes_alloc(w, split_nodeidv, split_nodev);
+        if (w->cw_err) {
+            kvdb_health_error(hp, w->cw_err);
+            goto done;
+        }
+    }
```

Comment:
Seems rather heavy handed to bring down the kvdb..  We do similar in cn_comp_compact() as well, longer term we need to try to stay online in the face transient errors and/or situations where data integrity is not in jeopardy...

Reply:
Agreed, removed the health error.